src/ImageList/index.js

```diff
@@ -54,10 +63,9 @@ export default class ImageList extends Component {
     return columns
   }
 
-  renderMasonry() {
+  renderMasonry(items) {
```

Comment:
It would actually probably be better to keep `this.getColumns`. Then here you'd pass in `items` from `renderMasonry` to `getColumns`